src/overlays/effects/ovl_Effect_Ss_Extra/z_eff_ss_extra.c

```diff
@@ -1,29 +1,94 @@
 /*
  * File: z_eff_ss_extra.c
  * Overlay: ovl_Effect_Ss_Extra
- * Description:
+ * Description: The floating points hovering over a target you hit in OOT
+ *   IE: 30/60/100 points when hitting the target in lost woods with slingshot to get bigger seed sack
  */
 
 #include "z_eff_ss_extra.h"
+#include "objects/object_yabusame_point/object_yabusame_point.h"
 
 #define PARAMS ((EffectSsExtraInitParams*)initParamsx)
 
-s32 EffectSsExtra_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
+u32 EffectSsExtra_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
 void EffectSsExtra_Update(GlobalContext* globalCtx, u32 index, EffectSs* this);
 void EffectSsExtra_Draw(GlobalContext* globalCtx, u32 index, EffectSs* this);
 
-#if 0
+static s16 gScores[] = { 30, 60, 100 };
+
 const EffectSsInit Effect_Ss_Extra_InitVars = {
     EFFECT_SS_EXTRA,
     EffectSsExtra_Init,
 };
 
-#endif
+static u64* gPointTextures[] = { gYabusamePoint30Tex, gYabusamePoint60Tex, gYabusamePoint100Tex };
+
+#define rObjId regs[0]
+#define rTimer regs[1]
+#define rScoreIdx regs[2]
```

Comment:
```suggestion
#define rScoreIndex regs[2]
```

Reply:
Right. I know OoT is in the process of changing `Idx` -> `Index`, it's probably just a matter of this file hasn't been looked at yet since the priority has been the engine